docs/open-targets.md

```diff
@@ -0,0 +1,59 @@
+  * *Benign*: it has been established that the variant is *not* implicated in this disease
+  * *Likely pathogenic* and *Likely benign* have the same meaning as “Pathogenic” and “Benign”, but signify that the connection isn't as certain (usually due to limited information being available for this variant)
+  * *Uncertain significance* means that either there is contradictory information on the impact of this variant, or that information is insufficient to draw the conclusion
+* **Review status** shows how many submitters provided information for this variant. Example values:
+  + no assertion criteria
+  + criteria provided, single submitter
+  + criteria provided, multiple submitters, no conflicts
+
+ClinVar is continuously updated and holds monthly releases of its database contents.
+
+## OpenTargets
+[OpenTargets](https://www.opentargets.org/) is a collaboration between academia and industry. Among other things, it combines associations between genetic variation and human traits (most notably, diseases) into a single integrated resource. This information is then used to provide evidence on the biological validity of therapeutic targets and an initial assessment of the likely effectiveness of pharmacological intervention on these targets.
+
+OpenTargets also holds periodic releases, which happen approximately every two months. Data for every release comes from several data providers. There are several requirements for submitting data to OpenTargets:
+* It must be represented in the form of “evidence strings”. These are JSON strings describing:
+  + Genes the variant affects
+  + Functional consequence of the variant on the gene
+  + Traits (usually diseases) associated with the variant, such as “parkinson disease” or “age-related macular degeneration”.
+  + Other information about the variant and source, such as related publications
+* The variant data must be synchronised with a specific version of external data sources, for example Ensembl.
+
+## Role of EVA
+ClinVar data is highly valuable, but in its original form is not suitable for submission to OpenTargets. EVA is registered as one of the submitters for OpenTargets. For every OpenTargets release, the EVA processes ClinVar records (variants), curates the result and submits it to OpenTargets in the form of evidence strings. This allows for the up-to-date ClinVar data to be integrated into the OpenTargets platform.
+
+Approximately one month before the submission deadline, OpenTargets will contact their submitters and specify the requirements for the next release. At this point the EVA can start executing the main submission protocol (see below). Once the data is ready, it is submitted to OpenTargets, and then the same will happen with the next release. Most of the actions in the pipeline are automated.
+
+
+
+# Workflow diagram
+
+![](docs/workflow-diagram/workflow.png)
+
+See details [here](docs/workflow-diagram) about how to regenerate this diagram from source.
```

Comment:
```suggestion
See details [here](./workflow-diagram) about how to regenerate this diagram from source.
```
I think as per my comment above, the "docs" is an extra and should be removed

Reply:
I actually just included the diagram directly in the file, thanks to the github mermaid magic you showed me :mermaid:

Reply:
🧜‍♂️ for the win